src/components/numeral-date/numeral-date.component.tsx

```diff
@@ -170,6 +170,9 @@ const validationMessages = (
 });
 
 const getDaysInMonth = (month?: string, year?: string) => {
+  if (month && (+month > 12 || +month < 1)) {
+    return 31;
+  }
```

Comment:
**question**: I'm not sure I fully understand what this guard is doing, may I ask what the reason is behind it so I can understand better?